### DISCUSSION TITLE
[Explore] Display log options in a collapsible component

### DISCUSSION THREAD

**Discussion by svennergr** [grafana/grafana#63005] [Observability]
Log options in Explore are currently displayed like this:
![image](https://user-images.githubusercontent.com/8092184/217218992-2dd57609-6b10-43cb-877b-99ba4d22f2a0.png)

By this it's hard to get descriptions about the meaning of those options, e.g. what is "Dedup" exactly doing. Additionally the existing options already take up a good amount of horizontal space.
The idea here would be to create a collapsible component, which holds the options. 
All options are visible and have some kind of description (e.g. via tooltips) when the panel is expaned, only some options/states are visible when the panel is collapsed.

Related issues:
- https://github.com/grafana/grafana/pull/62941#issuecomment-1419427488

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82863. Feel free to continue the discussion around this there. Thank you!
